app/views/idv/sessions/index.html.slim

```diff
@@ -5,47 +5,47 @@
   h1.heading = t('idv.titles.session.basic')
   .mb3
     label = t('idv.form.first_name')
-    = block_text_field_tag 'first_name', nil
+    = block_text_field_tag 'first_name', nil, required: true
     label = t('idv.form.last_name')
-    = block_text_field_tag 'last_name', nil
+    = block_text_field_tag 'last_name', nil, required: true
   h1.heading = t('idv.titles.session.dob')
   .mb3
     label.hide = t('idv.form.dob')
-    = block_date_field_tag 'dob', nil
+    = block_date_field_tag 'dob', nil, required: true
   h1.heading = t('idv.titles.session.ssn')
   .mb3
     label.hide = t('idv.form.ssn')
-    = block_text_field_tag 'ssn', nil
+    = block_number_field_tag 'ssn', nil, required: true
```

Comment:
Number fields do not work for zip, SSN, account numbers, since the fields are really numeric strings, not numbers. (At least on firefox, numeric fields add adjustment buttons which do not work as expected. For example, try a zip with a leading 0 and then use the adjustment button). I believe we need Javascript and/or Rails validation for these fields.